recipes/cylc-flow/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "cylc-flow" %}
+{% set version = "8.0a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
```

The only place that quotes are needed is here to prevent YAML interpreting it as a float and `8.10` being confused with `8.1`:
```suggestion
  version: "{{ version }}"
```

---

Thanks! I learnt something new today.

---

Understood, thanks.